radar/languages_frameworks/jaeger.yml

```diff
@@ -0,0 +1,13 @@
+version: 1
+name: Jaeger
+blip:
+  - version: "2.6"
+    ring: ADOPT
```

Comment:
Adopt - Is this used in production already? Proven at Extenda Retail?

Reply:
It is, however it might not be configured in any production environments. The packages and code are however in production.